internal/model.go

```diff
@@ -73,15 +73,15 @@ func (sd *SubscriptionData) Clone() *SubscriptionData {
 
 	if sd.Tags.Items() != nil {
 		cloned.Tags = utils.NewSet()
-		for _, value := range sd.Tags.Items() {
-			cloned.Tags.Add(value)
+		for key, value := range sd.Tags.Items() {
+			cloned.Tags.AddKV(key, value.UniqueID())
```

Comment:
why value become uniqueId? @cserwen

Reply:
`Key` is the real tag.
![image](https://user-images.githubusercontent.com/46882838/234832061-9ee27e80-387b-41e3-ae2a-2a580af64fb2.png)